src/screens/Exchange/Exchange.js

```diff
@@ -339,9 +340,10 @@ class ExchangeScreen extends React.Component<Props, State> {
     this.emptyMessageTimeout = null;
   };
 
-  checkIfAssetsExchangeIsAllowed = () => {
-    const { accounts, smartWalletState, isActiveAccountSmartWallet } = this.props;
-    if (!isActiveAccountSmartWallet) return true;
+  isAssetsExchangeAllowed = (): boolean => {
+    const { accounts, smartWalletState, activeAccount } = this.props;
+    if (!isArchanovaAccount(activeAccount)) return true;
```

Comment:
Are we not allowing exchange for etherspot atm?

Reply:
We are, this is a check `if there's no Archanova account then it returns true, otherwise if it's Archanova then there's additional check below if Archanova is deployed`.